CHANGELOG.md

```diff
@@ -73,6 +73,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#1870](https://github.com/nix-rust/nix/pull/1870))
 - The `length` argument of `sys::mman::mmap` is now of type `NonZeroUsize`.
   ([#1873](https://github.com/nix-rust/nix/pull/1873))
+- The MSRV is now 1.63
```

Comment:
Arg, you need to move this part to the top of the file.

Reply:
Shucks, fixed.